source/includes/mongodb-compatibility-table-csharp.rst

```diff
@@ -14,199 +15,195 @@
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
    * - 2.9
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
    * - 2.8
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
    * - 2.7
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
    * - 2.6
-     -
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
    * - 2.5
-     -
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
    * - 2.4
-     -
-     -
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
+     - ✓
    * - 2.3
-     -
-     -
-     -
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
    * - 2.2
-     -
-     -
-     -
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
    * - 2.0
-     -
-     -
-     -
-     -
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
    * - 1.11
-     -
-     -
-     -
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
-
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓
+     - ✓
    * - 1.10
-     -
-     -
-     -
-     -
-     -
-     -
-     -
-     -
-     - |checkmark|
-     - |checkmark|
-
-.. include:: /includes/older-server-versions-unsupported.rst
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+     - ✓
+     - ✓.. include:: /includes/older-server-versions-unsupported.rst
```

Comment:
```suggestion
     - ✓
    
   .. include:: /includes/older-server-versions-unsupported.rst
```

Reply:
Sorry I missed that, thanks for catching.